README.md

```diff
@@ -149,11 +145,12 @@ module "network" {
 module "loadbalancer" {
   source = "Azure/loadbalancer/azurerm"
   resource_group_name = "${var.resource_group_name}"
-  location = "${var.location}"
-  prefix = "terraform-test"
-  "lb_port" {
-      http = [ "80", "Tcp", "80"]
-  }
+  location            = "${var.location}"
```

Comment:
Do you have to make the same changes in test fixtures ? As I remember, I had taken the example from the readme files. Check if the test passes by running manually local.

Reply:
The build tests DO pass.  https://travis-ci.org/Azure/terraform-azurerm-computegroup/jobs/333506833

It's just the e2e tests which fail.  If you have a solution to make the e2e tests work, I'm all ears.